src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Boot.Server.ts

```diff
@@ -38,8 +40,18 @@ function boot() {
     .catch(unhandledError);
 }
 
-function unhandledError() {
-  err => console.error(err);
+function unhandledError(err) {
+  console.error(err);
+
+  // Disconnect on errors.
+  //
+  // TODO: it would be nice to have some kind of experience for what happens when you're
+  // trying to interact with an app that's disconnected.
```

Comment:
For RTM I agree, but I think having errors show up in the console is fine for 0.5.0.